sycl/include/sycl/handler.hpp

```diff
@@ -2364,11 +2367,15 @@ class __SYCL_EXPORT handler {
 
   template <typename KernelName = detail::auto_name, typename PropertiesT,
             typename... RestT>
-  std::enable_if_t<
-      (sizeof...(RestT) > 1) &&
-      detail::AreAllButLastReductions<RestT...>::value &&
-      ext::oneapi::experimental::is_property_list<PropertiesT>::value>
-  parallel_for(range<1> Range, PropertiesT Properties, RestT &&...Rest) {
+  __SYCL2020_DEPRECATED(
```

Comment:
A new test should be added into `sycl/test/warnings/` to check that deprecated message is correct and emitted for all overloads